### PR TITLE
Add ability to move a brief from unsuccessful to closed

### PR DESCRIPTION
**Pull request by tim-s-ccs**
Add route which will move the supplier from unsuccessful
This is required as we cannot update unsuccessful Briefs through the API

Add tests for the various scenarios that can occur when the request is made

This ticket was created due to a [request in the service desk](https://crowncommercial.zendesk.com/agent/tickets/34755)

### CODE REVIEW

File: app/main/views/briefs.py [hunk 1]
```diff
@@ -546,3 +546,37 @@ def list_brief_services(brief_id):
...
+    updater_json = validate_and_return_updater_request()
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if brief.status != 'unsuccessful':
+        abort(400, "Can only move an unsuccessful brief to close")
+
+    # Set status back to pending-awarded
```
Comment (bjgill) [inline app/main/views/briefs.py:566]: Is this true? It doesn't match with the method name and doc comment.

Otherwise, looks good.

Comment (tim-s-ccs) [reply app/main/views/briefs.py:566]: I'm not sure I understand what you mean? How does it not match the method name and doc comment, I'm not sure what I'm missing.

Comment (bjgill) [reply app/main/views/briefs.py:566]: Are we setting the status back to `pending-awarded` or `closed`? Or to those two mean the same thing?

Comment (bjgill) [reply app/main/views/briefs.py:566]: Just looked it up. `pending-awarded` is not a Brief status - it's a Brief Response status.

Comment (tim-s-ccs) [reply app/main/views/briefs.py:566]: Fortunately, we don't have to worry about brief responses with this as an unsuccessful brief will not have any

**Review by tberey — APPROVED** [review]

File: app/main/views/briefs.py (hunk 1 above)
Comment (bjgill) [reply app/main/views/briefs.py:566]: Sorry - I should have been clearer. 

```suggestion
    # Set status back to closed
```

**Pull request merged**